### ISSUE TITLE
[question] Release Date for V1.0

### ISSUE THREAD

**Issue by CodeMeZone** [iqltechnologies/brighty#2]
Is this is going to be a thing?

**Comment by tariq-abdullah** [comment]
Yes it will be a thing, god willingly. I had a change of mind regarding making the code available open-source. 
but now I am convinced Open source is the way. You will soon see project activity on this project.

**Comment by sotaspaso** [comment]
Any news on this project?
Thanks,
